README.md

```diff
@@ -1,63 +1,59 @@
 # MapleDiscBot
-"Universal" plug n play maplestory discord bot for your private server.
-Any questions or bug reports, my discord is Not Brandon#4444
+### "Universal" plug n play maplestory discord bot for your private server.
+If you have any questions or bug reports, my discord is **not** Brandon#4444
```

Comment:
I believe you misunderstood - his Discord ID is literally "Not Brandon#4444"
```suggestion
If you have any questions or bug reports, my Discord ID is `Not Brandon#4444`
```

Reply:
Oh man. I actually thought he didn't want people to contact him on Discord XD